.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,97 @@
+name: Publish Release
```

Comment:
This workflow is to be called from `main.yaml`, so remember to set that up too.

Reply:
done